### ISSUE TITLE
For emoji, allow us to specify the size at which they'll be displayed.

### ISSUE THREAD

**Issue by db0** [LemmyNet/lemmy#3520]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Is your proposal related to a problem?

Currently the uploaded emojis are scaled down to "emoji size" when displayed in lemmy-ui, but other clients display them at full size which is unwanted.

### Describe the solution you'd like.

I want to suggest to add a setting in the emojis, where the instance admin can choose the size they'll be displayed at. 

pict-rs' `/image/process` endpoint allows the use of `thumbnail={int}` and `resize={int}` which could be used when displaying an emoji to ensure it fits the dimentions wanted by the admins automatically, without requiring the admins to resize the images  manually before uploading

The display of emojis could be adjusted to take into account this setting, and retrieve from pict-rs using this processing.

### Describe alternatives you've considered.

Another alternative would be to handle the resize simply via the UI, which I think is happening already. Unfortunately most other clients don't know this setting, so there should be a way to inform them of the size of the emoji.

**Comment by Nutomic** [comment]
> Another alternative would be to handle the resize simply via the UI, which I think is happening already in the lemmy ui and jerboa.

So this is a problem in specific clients, nothing needs to be changed in the Lemmy backend. FYI emojis always look like this: `![downbear](https://lemmy.ml/api/v3/image_proxy?url=https%3A%2F%2Fhexbear.net%2Fapi%2Fv3%2Fimage_proxy%3Furl%3Dhttps%253A%252F%252Fchapo.chat%252Fpictrs%252Fimage%252F2ecd0805-fe83-4761-9771-8844f0e83a78.png "emoji downbear") `

So if inside `()` there is a part in quotes `""` its an emoji.

**Comment by dessalines** [comment]
Agree, this is a UI specific issue that should just use a sane default chosen by the UI designer, not a setting we need to add in the backend.
